Assignment/solution_student/CSS/style.css

```diff
@@ -0,0 +1,26 @@
+body{
+	margin-left: 60px;
+	margin-right: 60px;
+	margin-top: 60px;
+	margin-bottom: 60px;
+	background-color: white;
+	color: black;
+	text-align: center;
```

Comment:
I think this is throwing things off for you, and I think it's probably why you tried the `text-align: left;` on the `strong` element on [line 18](https://github.com/GA-Chicago-FEWD6/Week_01_Basics/pull/25/files#diff-b3e354f5b1e3f82611ec699aa0452c4aR18). It looks like you really only needed the title and the nav centered, so it would have been better to wrap those two things in a containing element like a `<header>`, and then apply `text-align: center;` to that header. Does that make sense? Something like this:
```
<header>		
	<h3><a href="about_me.html">About Me</a> | <em>Portofolio</em></h3>
	<h1>JOE JOHNSON</h1>
</header>
```
Though, while I'm at it, semantically it doesn't make sense for those nav links to be in an h3, which is intended for headings, so it would be better if those were wrapped in the `<nav>` element :)